### PR TITLE
Feat/mpp 515/phpcsfixer prestashop

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**
### Reason for change

[Linear task](https://linear.app/almapay/issue/MPP-LINEAR_ISSUE_ID)

### How to test

_As a reviewer, you are encouraged to test the PR locally._

### Checklist for authors and reviewers

- [ ] The title of the PR uses business wording, not technical jargon, for the changelog readers to understand it
- [ ] The PR implements the changes asked in the referenced task / issue
- [ ] The automated tests are compliant with the [testing strategy](https://www.notion.so/almapay/Backend-testing-strategy-06c642cec1bf47b9b8feca3a91ea8d4a)
- [ ] The tests are relevant, and cover the corner/error cases, not only the happy path
- [ ] You understand the impact of this PR on existing code/features
- [ ] The changes include adequate logging and Datadog traces
- [ ] Documentation is updated (API, developer documentation, ADR, Notion...)

### CODE REVIEW

File: alma/lib/Forms/AbstractAlmaAdminFormBuilder.php [hunk 1]
```diff
@@ -50,7 +48,7 @@ abstract class AbstractAlmaAdminFormBuilder extends AbstractAdminFormBuilder
      * @param string $image
      * @param array $config
      */
-    public function __construct(Alma $module, Context $context, $image, $config = [])
+    public function __construct($module, $context, $image, $config = [])
```
Comment (Francois-Gomis) [inline alma/lib/Forms/AbstractAlmaAdminFormBuilder.php:51]: You remove typing ?

File: alma/lib/Helpers/CarrierHelper.php [hunk 2]
```diff
@@ -37,12 +36,12 @@
...
 {
     const UNKNOWN_CARRIER = 'Unknown';
-    /** @var Context */
+    /** @var \Context */
     private $context;
     /** @var CarrierData */
     private $carrierData;
 
-    public function __construct(Context $context)
+    public function __construct(\Context $context)
```
Comment (Francois-Gomis) [inline alma/lib/Helpers/CarrierHelper.php:44]: Typing  or no typing ?

File: alma/lib/Helpers/OrderStateHelper.php [hunk 3]
```diff
@@ -35,15 +33,14 @@
  */
 class OrderStateHelper
 {
-    public function __construct(
-        Context $context
-    ) {
+    public function __construct($context)
```
Comment (Francois-Gomis) [inline alma/lib/Helpers/OrderStateHelper.php:36]: No typing for context

File: alma/tests/unit/lib/Model/CartDataTest.php [hunk 4]
```diff
@@ -21,13 +21,16 @@
  * @copyright 2018-2023 Alma SAS
  * @license   https://opensource.org/licenses/MIT The MIT License
  */
+
 namespace Alma\PrestaShop\Tests\Unit\Lib\Model;
 
 use Alma\PrestaShop\Helpers\ProductHelper;
 use Alma\PrestaShop\Model\CartData;
 use Alma\PrestaShop\Repositories\ProductRepository;
 use Cart;
```
Comment (Francois-Gomis) [inline alma/tests/unit/lib/Model/CartDataTest.php:30]: you have Cart not \Cart

**Review by Francois-Gomis — COMMENTED** [review]
ok

File: alma/lib/Helpers/CarrierHelper.php (hunk 2 above)
Comment (Benjamin-Freoua-Alma) [reply alma/lib/Helpers/CarrierHelper.php]: No typing done

**Review by Francois-Gomis — APPROVED** [review]

**Pull request merged**